d4s-test/src/main/scala/d4s/test/envs/DynamoTestEnv.scala

```diff
@@ -8,12 +8,12 @@ import izumi.distage.docker.Docker
 import izumi.distage.docker.modules.DockerContainerModule
 import izumi.distage.model.definition.DIResource
 import izumi.distage.testkit.TestConfig
-import izumi.distage.testkit.services.scalatest.dstest.DistageAbstractScalatestSpec
+import izumi.distage.testkit.scalatest.DistageBIOSpecScalatest
 import izumi.functional.bio.{BIOApplicative, F}
 import logstage.LogBIO
 import net.playq.aws.tagging.AwsNameSpace
 
-trait DynamoTestEnv[F[+_, +_]] extends DistageAbstractScalatestSpec[F[Throwable, ?]] {
+trait DynamoTestEnv[F[+_, +_]] extends DistageBIOSpecScalatest[F] {
```

Comment:
@aliubymov-playq 
In this case users won't be able to inherit `DistageBIOEnvSpecScalatest`
Try adding `implicit def tagBIO: TagKK[F]` here and inheriting `DynamoTestBase` from test spec later